app/controllers/api/api_controller.rb

```diff
@@ -1,4 +1,4 @@
-class Api::ApiController < ActionController::Base
+class Api::ApiController < ActionController::API
```

Comment:
@t-harps @GarySiu are there specific reasons to inherit this and below one from Base controller and not from API ?